app/assets/javascripts/discourse/app/components/post-text-selection.gjs

```diff
@@ -89,9 +86,20 @@ export default class PostTextSelection extends Component {
 
   @bind
   async selectionChanged() {
-    let supportsFastEdit = this.canEditPost;
-    const selection = window.getSelection();
+    const _selectedText = selectedText();
+
+    // avoid hard loops in quote selection unconditionally
+    // this can happen if you triple click text in firefox
+    // it's also generally unecessary work to go
+    // through this if the selection hasn't changed
+    if (this.menuInstance?.expanded && this.prevSelection === _selectedText) {
+      this.prevSelection = _selectedText;
```

Comment:
We just checked whether `this.prevSelection === _selectedText`... so is there any need for the assignment here? Seems like a no-op to me?

Reply:
yes right I should remove it :+1: